text/0000-cargo-libstd-awareness.md

```diff
@@ -0,0 +1,236 @@
+- Feature Name: cargo_stdlib_awareness
+- Start Date: 2015-05-26
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+
+# Summary
+
+Currently, Cargo doesn't know what parts of the standard library packages depend on.
+By giving it this knowledge, we can make cross compilation and exotic platform development easier, simplify rustbuild, and allow anyone to easily specify bounds on the standard library for their packages.
+This will allow building parts of the standard library from source, but in order to not disrupt existing workflows, the binaries that come with rustc will still be used by default.
+
+
+# Motivation
+
+First, some background.
+`rustc` can be given crates and their location with `--extern <name>=<path>`.
```

Comment:
Heh that old paragraph might be the oldest part of the RFC now. Might be easiest to rewrite after https://github.com/rust-lang/rfcs/pull/1133#discussion_r73763659 is worked out once and for all.